sdcm/sct_config.py

```diff
@@ -566,6 +566,10 @@ class SCTConfiguration(dict):
              help="Url to the repo of scylla manager version to upgrade to for management tests"),
 
         # PerformanceRegressionTest
+        dict(name="max_partitions_with_data_validate", env="SCT_MAX_PARTITIONS_WITH_DATA_VALIDATE", type=int,
+             help="""Relevant for scylla-bench. Hold maximum -partition-count that data is written with
```

Comment:
description is not clear for one who reads it and don't know what you know

Reply:
changed